tests/unit/voucher/AccountAuthorized.test.ts

```diff
@@ -14,8 +15,8 @@ const VOUCHER_DURATION = BigInt.fromI32(86400);
 const VOUCHER_TYPE_ELIGIBLE_ASSETS: string[] = [];
 
 const VOUCHER_OWNER = '0xabcdefabcdefabcdefabcdefabcdefabcdefabcd';
-const VOUCHER_VALUE = BigInt.fromI32(100);
-const VOUCHER_BALANCE = BigInt.fromI32(50);
+const VOUCHER_VALUE = BigDecimal.fromString('100');
```

Comment:
```suggestion
const VOUCHER_VALUE = BigDecimal.fromString('100.1');
```
Maybe put "real" decimals in tests to check everything goes well

Reply:
Good point, done thanks